util/terraform.go

```diff
@@ -34,22 +35,23 @@ type (
 
 var (
 	reTfVariables = regexp.MustCompile(`(?ms)^variable\s+".*?"\s+{(\s*\n.*?^}|\s*}$)`)
-	cacheDefault  = make(map[string]*cachedDefaultVariables)
+	cacheDefault  sync.Map
```

Comment:
It is always a bad idea to have a global map variables. That was making the tests flaky as we run many threads and we got random exceptions while running them. Always use a sync.Map to make sure that the map can be accessed/modified concurrently.